package.json

```diff
@@ -62,9 +62,9 @@
     "whatwg-fetch": "^3"
   },
   "dependencies": {
+    "@brightspace-ui/intl": "Brightspace/intl#v3",
```

Comment:
This still needs to be updated to the actual publish location on NPM when it's there.